### PR TITLE
Escape JS reserved words when generating the build-id

### PR DESCRIPTION
**Pull request by arichiardi**
The patch also re-introduces testing with alt-test.

I decided to open a PR so that folks can see and comment. I will squash merge if not objection is risen, especially by @Deraen :smile:.

### CODE REVIEW

File: src/powerlaces/boot_figreload.clj [hunk 1]
```diff
@@ -90,7 +90,7 @@
   the fileset."
   [tmpfiles]
   (mapv #(-> {}
-             (assoc :build-id (util/build-id (:path %)))
+             (assoc :build-id (-> % b/tmp-file .getName util/build-id))
```
Comment (Deraen) [inline src/powerlaces/boot_figreload.clj]: Hmm, why is only the name used? This way files would clash if there are e.g. a/main.cljs.edn and b/main.cljs.edn?

Comment (arichiardi) [reply src/powerlaces/boot_figreload.clj]: Uhm, right, so there can be two `build-id` with the same name. This concept is a bit different from what figwheel considers a build-id I guess? I will need to calculate it in a different way then, maybe hashing the path?

Comment (arichiardi) [reply src/powerlaces/boot_figreload.clj]: So yes, it is unique in Figwheel because it comes from `cljsbuild` ids.

Comment (arichiardi) [reply src/powerlaces/boot_figreload.clj]: Tried with hashing, it works, I will change this.

**Comment by arichiardi** [comment]
I fixed that, one thing to think about is how to handle multiple figwheel "builds" at the same time. I guess it will be in another patch.

**Comment by arichiardi** [comment]
@Deraen all good?

**Pull request merged**